cli/README.md

```diff
@@ -0,0 +1,52 @@
+# Cobalt CLI
+
+Currently this only contains the start of the project structure, a test concept, Makefile and Readme. From a terminal, cd into the `cli` folder and run:
+
+```make deps```
+```make build```
+```make run```
+```make test```
+
+We are currently only using go-cobra and go-testing packages just to get something committed. 
+
+# Cobalt CLI Commands
+
+List help commands:
+
+```cobalt --help```
+
+Setup template in current directory:
+
+```cobalt setup --template "bedrock/azure-simple"```
+
+> The flag `--template` is specified using the following structure:
+>
+> `[github repo name]/[cobalt template name]`
+
+Test the current template in current directory:
+
+```cobalt run --docker```
```

Comment:
Can you elaborate on what will be run when this command is implemented? Does this actually build out resources?

Looking down further I see that it is a test runner. Should this be called something along the lines of `test`?

Reply:
Should be an easy refactor and I don't disagree. @erikschlegel for context...

It does nothing no so the docs, ATM, are shallow. I've not worked out the spec for the story yet so not really sure :-P